packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -303,9 +302,14 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
     />
   );
 
+  useEffect(() => {
+    if (containerRef.current) {
+      containerRef.current.id = `video-gallery-${uuidv4()}`;
```

Comment:
The idea is that whenever our containerRef changes we want to use a new UUID so we don't have collisions between different video gallery instances?

Reply:
curious to know too

Reply:
This useEffect only happens since containerRef is instantiated only once during the lifetime of the VideoGallery component. So the UUID is only created once so it should be the same for a VIdeoGallery's lifetime.

Reply:
Why not simply generate an `id` once.

    containerId = useRef(`video-gallery-${uuidv4()}`)
    ...

    return (<div id={containerId.current}> .... </div>)

Reply:
I like that better too. Changed. Thanks.